junit-runtime/src/main/scala/org/scalajs/junit/JUnitExecuteTest.scala

```diff
@@ -199,8 +199,12 @@ final class JUnitExecuteTest(taskDef: TaskDef, runner: JUnitBaseRunner,
 
   private[this] def logFormattedInfo(method: String, msg: String): Unit = {
     val fMethod = if (method != null) c(method, NNAME2) else null
-    richLogger.info(
-        formatLayout("Test ", packageName, c(className, NNAME1), fMethod, msg))
+    val formatted = formatLayout("Test ", packageName, c(className, NNAME1),
+        fMethod, msg)
+    if (verbose)
+      richLogger.info(formatted)
+    else
+      richLogger.debug(formatted)
```

Comment:
Actually it is only for the `started` and `ignored` ones. I should change the name of the function or add another level of indirection. The implementation follows the original implementation rather than the documentation to have a similar output (modulo stack traces).